leap_c/examples/hvac/env.py

```diff
@@ -561,7 +561,26 @@ def reset(
         else:
             min_start_idx = 0
             max_start_idx = len(self.data) - self.N_forecast - self.max_steps + 1
-            self.idx = self.np_random.integers(low=min_start_idx, high=max_start_idx + 1)
+
+            self.idx = min_start_idx
+
+            n_max = 1000
+            counter = 0
+            date_valid = False
+            while not date_valid and counter < n_max:
+                idx = self.np_random.integers(low=min_start_idx, high=max_start_idx + 1)
```

Comment:
Do you know where this object `self.np_random` is created and seeded? I only know that `gym.Env.reset(seed=seed)` seeds `self._np_random`.